src/tsf/TfConvArea.cpp

```diff
@@ -44,15 +44,13 @@ HRESULT CConversionArea::DrawComposition(const CComBSTR& CompStr,
                                                                  DEFAULT_COMP_INPUT_ERROR
                                                                };
 
-    std::wstring_view text(CompStr, CompStr.Length());
-
     const auto encodedAttributes = _DisplayAttributesToEncodedAttributes(DisplayAttributes,
                                                                          CompCursorPos);
 
     std::basic_string_view<BYTE> attributes(encodedAttributes.data(), encodedAttributes.size());
     std::basic_string_view<WORD> colorArray(colors.data(), colors.size());
 
-    return ImeComposeData(text, attributes, colorArray);
+    return ImeComposeData(CompStr, attributes, colorArray);

```

Comment:
Would be nicer to rename the parameter to `text` instead

Reply:
I don't think we should rename it. It has it's meaning.